app/(app)/tools/body-scan/julia.tsx

```diff
@@ -0,0 +1,36 @@
+import useTranslationKeys from '@/hooks/useTranslationKeys';
+import { Icon } from '@/components/Icon';
+
+import { ScreenWithImageHeader } from '@/components/ScreenWithImageHeader';
+import { Typography } from '@/components/Typography';
+import { Stack, useRouter } from 'expo-router';
+import { useTranslation } from 'react-i18next';
+import { useAssetsManagerContext } from '@/contexts/AssetsManagerContextProvider';
+
+export default function JuliaPreview() {
+  const router = useRouter();
+  const { t } = useTranslation('tools');
+
+  const { toolsTranslationKeys } = useTranslationKeys();
+  const { mediaMapping } = useAssetsManagerContext();
+
+  return (
+    <>
+      <Stack.Screen options={{ headerShown: false }} />
```

Comment:
because this route is already defined in the stack layout, we can remove this from here, because now we're overriding the definition of the screen